wallet/src/de/schildbach/wallet/ui/UnlockWalletDialogFragment.java

```diff
@@ -32,20 +32,24 @@
 
     private static final String FRAGMENT_TAG = UnlockWalletDialogFragment.class.getName();
 
-    public static void show(final FragmentManager fm) {
-        new UnlockWalletDialogFragment().show(fm, FRAGMENT_TAG);
+    public static UnlockWalletDialogFragment show(final FragmentManager fm) {
+        UnlockWalletDialogFragment unlockWalletDialogFragment = new UnlockWalletDialogFragment();
+        unlockWalletDialogFragment.show(fm, FRAGMENT_TAG);
+        return unlockWalletDialogFragment;
```

Comment:
Based on the work done in the Pay/Receive PR's, the `CheckPinDialog `will be replacing `UnlockWalletDialogFragment`

Also, depending on how secure we want this setting to be, it may be not be necessary to prompt the user for their PIN.

Reply:
@sambarboza in the following commit you can find an example on how to easily do this: 
https://github.com/dashevo/dash-wallet/pull/284/commits/4fdaf6c74a4f4f09541859d6218fe7cae2212c1c#diff-7de70db1c2c14b9b4f5b273bab274f82R68

Reply:
Done on https://github.com/dashevo/dash-wallet/pull/286/commits/e6c46b0be2f9fe3ed8a3f091a79391f181cdb6b5.
@HashEngineering , @tomasz-ludek